packages/force_update/lib/src/force_update_storage.dart

```diff
@@ -7,20 +7,20 @@ import 'package:version/version.dart';
 class ForceUpdateResult {
   const ForceUpdateResult({
     required this.versionAtTimeOfRequest,
-    required this.result,
+    required this.conclusion,
   });
 
   ForceUpdateResult.fromJson(Map<String, dynamic> json)
       : versionAtTimeOfRequest =
             Version.parse(json['versionAtTimeOfRequest'] as String),
-        result = VersionSupportResultDTO.values[json['result'] as int];
+        conclusion = VersionSupportResultDTO.values[json['conclusion'] as int];
```

Comment:
this is somewhat breaking, but the former name does not make sense, it caused some weird expressions like `forceUpdateResult.result`. So I am convinced that this should be changed, but I am not sure how. Should I handle reading json differently if there is a result stored by a previous version of the package? Have we even used this package anywhere yet?